src/components/interaction-designer/block-editors/GroupMembershipEditor.vue

```diff
@@ -34,6 +34,7 @@
             :options="groupOptions"
             :taggable="!availableGroups"
             :multiple="true"
+            :show-labels="false"
```

Comment:
fix for these 'hints' overlaying:
<img width="341" alt="Screenshot 2023-06-13 at 3 57 18 PM" src="https://github.com/FLOIP/flow-builder/assets/18130716/57977685-9c14-4f10-8266-6c6eab10eb52">
<img width="333" alt="Screenshot 2023-06-13 at 3 57 33 PM" src="https://github.com/FLOIP/flow-builder/assets/18130716/e3f50d90-2801-4155-b3fc-2bcba223e3d2">